internal/config/config.go

```diff
@@ -59,6 +59,7 @@ type Config struct {
 	DevelopersBannerLink     string
 	DevelopersBannerText     string
 	IsLocal                  bool
+	UrlProtocol              string
```

Comment:
can you please rename UrlProtocol to URLProtocol everywhere to keep with current naming conventions?